wordpress-mobile/WordPress-iOS
# Remove a couple of API request methods from MediaService

This PR removes two functions in `MediaService`. The removed functions are pure API requests, without accessing Core Data. They don't need to exist in `MediaService`, which typically uses Core Data (it's a subclass of `LocalCoreDataService` after all).

The removed functions' call sites now bypass `MediaService`, and use `MediaServiceRemote` to send API requests directly.

## Test Instructions

To test the "media library count" function, tap the site icon and select "Choose Image From My Device". Check if the media library count displayed is correct.

To test the "video press metadata" function, upload a video in post editor and check if the video is uploaded and displayed correctly.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
What's described in the Test Instructions

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:


UI Changes testing checklist: N/A
